packages/manager/modules/config/src/index.ts

```diff
@@ -20,6 +20,15 @@ export { LANGUAGES } from './locale/locale.constants';
 
 export const isTopLevelApplication = () => window.top === window.self;
 
+export const getAuthenticationURL = () => {
```

Comment:
**thought:** This logic looks quite similar to https://github.com/ovh/manager/blob/master/packages/components/ovh-reket/src/redirections.js

**question:** Is there any plan to add some refactor around it?

Reply:
Could be a good point to share this logic at the same place

Reply:
good point i'll see what i can do to reuse it